presto-sqlserver/src/main/java/io/prestosql/plugin/sqlserver/SqlServerClient.java

```diff
@@ -73,6 +77,22 @@ public void renameColumn(JdbcIdentity identity, JdbcTableHandle handle, JdbcColu
         }
     }
 
+    @Override
+    public PreparedStatement buildSql(ConnectorSession session, Connection connection, JdbcSplit split, List<JdbcColumnHandle> columnHandles)
+            throws SQLException
+    {
+        return new SqlServerQueryBuilder().buildSql(
```

Comment:
Now the `identifierQuote` selection logic is in 2 places:
- `SqlServerQueryBuilder::new`
- `SqlServerClient::new`

IMO it would be better to pass `this.identifierQuote` to the QueryBuilder constructor